workflow/rocoto/gfs_tasks.py

```diff
@@ -796,6 +796,66 @@ def postsnd(self):
 
         return task
 
+    def fbwinds(self):
+
+        def _get_fbwindsgroups(cdump, config):
+
+            fhmin = config['FHMIN']
+            fhmax = config['FHMAX']
+            fhout = config['FHOUT']
+
+            # Get a list of all forecast hours
+            fhrs = []
+            if cdump in ['gdas']:
```

Comment:
`fbwinds` only runs for `gfs`.

Reply:
@WalterKolczynski-NOAA Can you please explain what and why this was referenced? What is needed here?

https://github.com/NOAA-EMC/global-workflow/blob/62d8e99e77ffe44784f4c61c3f2296c6d1a72413/scripts/exgfs_atmos_fbwind.sh#L39C1-L51C5